logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/UserScope.kt

```diff
@@ -28,15 +28,15 @@ class UserScope(
     private val syncManager: SyncManager,
     private val assetRepository: AssetRepository,
     private val teamRepository: TeamRepository,
-    private val connectionRepository: ConnectionRepository,
+    private val connectionRepository: ConnectionRepository
```

Comment:
We can change this `UserScope` to fix the compilation error:

```kt
class UserScope internal constructor(
    private val userRepository: UserRepository,
    private val searchUserRepository: SearchUserRepository,
    private val syncManager: SyncManager,
    private val assetRepository: AssetRepository,
    private val teamRepository: TeamRepository,
    private val connectionRepository: ConnectionRepository
)
```

Reply:
done